ios/RNSScreenStackHeaderConfig.mm

```diff
@@ -1,7 +1,15 @@
-#import "RNSScreenStackHeaderConfig.h"
-#import "RNSScreen.h"
-#import "RNSSearchBar.h"
-
+#ifdef RN_FABRIC_ENABLED
+#import <React/RCTConversions.h>
+#import <React/RCTFont.h>
+#import <React/UIView+React.h>
+#import <react/renderer/components/rnscreens/ComponentDescriptors.h>
+#import <react/renderer/components/rnscreens/EventEmitters.h>
+#import <react/renderer/components/rnscreens/Props.h>
+#import <react/renderer/components/rnscreens/RCTComponentViewHelpers.h>
+#import "RCTFabricComponentsPlugins.h"
```

Comment:
Are all those imports needed? e.g. in `RNSScreen.mm` there are less of them and it seems to work fine.

Reply:
All the imports you highlighted are also present in `RNSScreen.mm`